venafi/resource_venafi_certificate.go

```diff
@@ -215,8 +215,8 @@ func resourceVenafiCertificateCreate(ctx context.Context, d *schema.ResourceData
 
 func resourceVenafiCertificateRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 
-	cfg := meta.(*vcert.Config)
-	cl, err := vcert.NewClient(cfg)
+	vCertCfg := meta.(*venafiProviderConfig).vCertCfg
```

Comment:
Possible NPE, if the casted object is nil or not the interface you are expecting

Reply:
Done.